library/CHART-TEMPLATE/tests/common_cluster_test.yaml

```diff
@@ -4,20 +4,40 @@ templates:
 release:
   name: my-release
   namespace: my-namespace
-set:
+
+# The name of the cluster is mandatory, to not make all the tests fail because they have no license
+# and not having to add license to all the tests I have added it to the `values.yaml`.
+# This change makes this tests a bit funky because we have to manually set local cluster to null
+#constantly to overwrite it.
```

Comment:
missing space after #

Reply:
Solved in [#ddaebab](https://github.com/newrelic/helm-charts/pull/759/commits/ddaebabdcb252bc5da99b20d1bf96e3f2f172e0f)